auth/utils.py

```diff
@@ -0,0 +1,15 @@
+import jwt
+
+SECRET_KEY = '1234567890'  # TODO: Move to environment variables
+
+def is_valid_password(password):
+    return len(password) > 5
```

Comment:
should be longer than 12
```suggestion
def is_valid_password(password):
    return len(password) > 12
```